src/appmixer/asana/tasks/ListUsers/ListUsers.js

```diff
@@ -18,15 +18,23 @@ module.exports = {
 
         let client = commons.getAsanaAPI(context.auth.accessToken);
 
-        return client.users.findByWorkspace(workspace)
-            .then(res => {
-                return commons.sendArrayOutput({
-                    context,
-                    outputPortName: 'users',
-                    outputType,
-                    records: res.data
-                });
-            });
+        let users;
+
+        try {
+            const res = await client.users.findByWorkspace(workspace);
+            users = res.data;
+        } catch (err) {
+            if (!context.properties.ignoreErrors) {
+                throw err;
+            }
+        }
```

Comment:
What kind of errors does it throw?

I would like to see some kind of error logging here even when we are ignoring the errors.

Reply:
I think this is not necessary as this is for the UI only, I think you really don't case about the errors, If you do care, you can use the ListUsers without the ignoreErrors flag